src/NuGet.Clients/PackageManagement.PowerShellCmdlets/Exceptions/PackageSourceException.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NuGet.PackageManagement.PowerShellCmdlets.Exceptions
```

Comment:
keep the root namespace `NuGet.PackageManagement.PowerShellCmdlets`, that's the team convention.